test/unit/mysql-helper.test.js

```diff
@@ -8,7 +8,7 @@
 const test = require('node:test')
 const assert = require('node:assert')
 
-const useParser = require('../../lib/db/utils').extractDatabaseChangeFromUse
```

Comment:
i would move this to a more relevant place.  `test/unit/lib/subscribers/mysql/helper.test.js`